clang/lib/Sema/SemaStmt.cpp

```diff
@@ -2783,6 +2789,7 @@ StmtResult Sema::BuildCXXForRangeStmt(SourceLocation ForLoc,
       LoopVar->setType(SubstAutoTypeDependent(LoopVar->getType()));
     }
   } else if (!BeginDeclStmt.get()) {
+
```

Comment:
Spurious whitespace change can be removed.

Reply:
removed